website/package.json

```
@@ -19,7 +19,9 @@
     "@docusaurus/preset-classic": "^2.0.0-alpha.56",
```

```suggestion
    "@docusaurus/preset-classic": "^2.0.0-alpha.59",
```
I think it might be better to update here as well based on your commit log.